### PR TITLE
Map Updates

### PR DESCRIPTION
**Pull request by Michab02**
Updated map, altered area files, slightly modified the stacking
machine's code.

### CODE REVIEW

File: code/game/area/Space Station 13 areas.dm [hunk 1]
```diff
@@ -2079,10 +2087,9 @@ area/space/atmosalert()
 			sound_delay = rand(0, 50)
 
 		for(var/mob/living/carbon/human/H in src)
-		//	if(H.s_tone > -55)
-		//		H.s_tone--
-		//		H.update_body()
-		//Goodbye Skintone, Orbis
+			if(H.s_tone > -55)
+				H.s_tone--
```
Comment (ShadowLarkens) [inline code/game/area/Space Station 13 areas.dm]: Rogue changes?

Comment (Michab02) [inline code/game/area/Space Station 13 areas.dm]: I definitely don't recognize that change, so possibly.

Why is there something about skin tones in the areas file anyway?

Comment (ShadowLarkens) [inline code/game/area/Space Station 13 areas.dm]: Who knows, really. Seems to be part of the beach area.

Comment (Michab02) [inline code/game/area/Space Station 13 areas.dm]: Oooh. I bet someone tried to make it so the beach gives you a tan.

Comment (Nienhaus) [inline code/game/area/Space Station 13 areas.dm]: They did, it's pretty cool.

Comment (SpadesNeil) [inline code/game/area/Space Station 13 areas.dm]: for(var/mob/living/carbon/human/H in src)
-		//	if(H.s_tone > -55)
-		//		H.s_tone--
-		//		H.update_body()
-		//Goodbye Skintone, Orbis
+			if(H.s_tone > -55)
+				H.s_tone--

Why is this removed?

Comment (ShadowLarkens) [inline code/game/area/Space Station 13 areas.dm]: Also, just a minor thing, not especially important for a non-major community I guess; but I don't think mapmerge was used for these changes.

Comment (Michab02) [inline code/game/area/Space Station 13 areas.dm]: I admittedly have no idea how to use mapmerge.

Also, what changes did Orbis make that were reverted?

**Comment by Michab02** [comment]
Okay, fixed everywhere where I overwrote Orbis' changes.

File: VOREStation.dme [hunk 2]
```diff
@@ -5,6 +5,72 @@
...
+#define FILE_DIR "icons/obj/assemblies"
+#define FILE_DIR "icons/obj/atmospherics"
+#define FILE_DIR "icons/obj/clothing"
+#define FILE_DIR "icons/obj/clothing/species"
+#define FILE_DIR "icons/obj/clothing/species/akula"
+#define FILE_DIR "icons/obj/clothing/species/sergal"
+#define FILE_DIR "icons/obj/clothing/species/skrell"
+#define FILE_DIR "icons/obj/clothing/species/tajaran"
+#define FILE_DIR "icons/obj/clothing/species/unathi"
+#define FILE_DIR "icons/obj/doors"
```
Comment (ShadowLarkens) [inline VOREStation.dme]: You should go to "Build - Preferences" and untick "Automatically set FILE_DIR for sub-directories"

Comment (Michab02) [inline VOREStation.dme]: Noted, thanks. I was wondering what was up with that.

File: code/game/area/Space Station 13 areas.dm (hunk 1 above)
Comment (OrbisAnima) [inline code/game/area/Space Station 13 areas.dm]: We could set skins colors 10-20 units towards black,if you still want the tanning effect, but we should be carefull to only apply that to non fur races (here comes a bit of ignorance on my part... do scales tan?)

**Comment by SpadesNeil** [comment]
So is that everything we need to worry about?

**Comment by Michab02** [comment]
To the best of my knowledge, yes!
